core/platform/lf_arduino_support.c

```diff
@@ -33,6 +33,11 @@ THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "../platform.h"
 #include "Arduino.h"
 
+/**
+ * Keep track of interrupts being raised.
+ */
+bool _lf_timer_interrupted;
```

Comment:
I don't see where this will ever get set to false after once being set to true. Won't the timer always think it has been interrupted after the first time?

Reply:
It's happening in `_lf_ack_events`.